SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1842,6 +1868,24 @@ bool USpatialNetDriver::CreateSpatialNetConnection(const FURL& InUrl, const FUni
 	return true;
 }
 
+void USpatialNetDriver::CleanUpConnection(USpatialNetConnection* ConnectionCleanedUp)
```

Comment:
```suggestion
void USpatialNetDriver::CleanUpClientConnection(USpatialNetConnection* ClientConnection)
```
I believe this is better naming.